packages/jest-cli/src/init/__tests__/fixtures/has_jest_config_file_ts/jest.config.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default {};
```

Comment:
these `init` stuff, should we generate `jest.config.ts` file if the user says "Yes" to use TypeScript?

Reply:
@SimenB Yes you're right, I will commit this later today.